{"name": "Тест на получение ошибки при попытке создания курьеров с одинаковыми логинами", "status": "passed", "description": "Создаем последовательно двух курьеров передавая одинаковые логины и проверяем, что возвращается ошибка", "start": 1745072125078, "stop": 1745072125344, "uuid": "d1615eb1-ded4-4379-8e32-f981c25c0b7b", "historyId": "b0188f64bd1dcf0f2ef38757ee9d9fab", "testCaseId": "b0188f64bd1dcf0f2ef38757ee9d9fab", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_duplicate_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "LAPTOP-3QFAQEQR"}, {"name": "thread", "value": "10504-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}